orders_service.py

```diff
@@ -28,7 +34,103 @@ def validate_order(self):
     
     def save_order(self):
         order_model = OrdersModel(name=self.user.name, birthday=self.user.birthday, email=self.user.email, state=self.user.state, zipcode=self.user.zipcode)
-        OrdersModel.save(order_model)
+        order_model.save()
+
+    def to_dict(self):
```

Comment:
Why do you need this method at order_server level, you have already implemented this at ordermodel level right >